sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -1539,7 +1567,7 @@ export class CertificateClient {
    *   subject: "cn=MyCert"
    * });
    * const poller = await client.getCertificateOperation("MyCertificate");
-   * const { csr } = poller.getResult();
+   * const { csr } = poller.getPublicState().operation!;
```

Comment:
Are we assuming operation is always defined?

Reply:
operation can't be defined if we get a server error from the start.